build/automation/var/profile/local.mk

```diff
@@ -32,3 +32,8 @@ NAME_PUBLIC_PRIORITY := 0
 # Service Data files
 SERVICE_DATA_FILE := create_all_services_local.sh
 LOCATIONS_DATA_FILE := load_locations_test.sh
+
```

Comment:
We also need to add this configuration the dev profile file? Or is it already there?

Reply:
this is just the local changes which I have made. I will create a different task to update all the other profile updates.